test/service/messageutils.js

```diff
@@ -105,4 +85,65 @@ describe("service/messageutils", () => {
       });
     });
   });
+
+  describe("sendNotificationToReceivers", () => {
+    it("should send a message to each receiver", () => {
+      const testClient = {
+        chat: {
+          postMessage: sinon.stub(),
+        },
+      };
+      const testGratitude = {
+        receivers: [
+          {
+            id: "testuser1",
+          },
+          {
+            id: "testuser2",
+          },
+        ],
+      };
+      sinon
+        .stub(recognition, "receiverSlackNotification")
+        .returns({ test: "test" });
+      return expect(
+        messageutils.sendNotificationToReceivers(testClient, testGratitude)
+      ).to.eventually.be.fulfilled.then(() => {
+        // Checks that the function passed to it was called twice (specifically 'testClient.chat.postMessage')
+        // This means sendNotificationToReceivers sent a message to each of the two receivers in the test.
+        sinon.assert.calledTwice(testClient.chat.postMessage);
+        sinon.assert.calledWith(testClient.chat.postMessage, {
+          channel: testGratitude.receivers[0].id,
+          text: "You earned a :fistbump:.",
+          test: "test",
+        });
+        sinon.assert.calledWith(testClient.chat.postMessage, {
+          channel: testGratitude.receivers[1].id,
+          text: "You earned a :fistbump:.",
+          test: "test",
+        });
+      });
+    });
+  });
+
+  describe("getRecieverMessage", () => {
+    it("should get golden fistbump message", async () => {
```

Comment:
I know this was just moved lower in the file, but there's no need for either this test or the one below to be `async`. There's no `await` in the test function or the production code.